### ISSUE TITLE
Improved Accessibility

### ISSUE THREAD

**Issue by dave-agilepixel** [elementor/elementor#9333]
## Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**

The Syntax of the output from the page builder is not broadly accessible, simple use of incorrect markup for accessibility, missing aria labelling, enforcing user habits to provide best practices for image tag, heading hierarchy.

**Describe the solution you'd like**
Improve and add accessiibilty to the output syntax in HTML to best suit the various accessibilty requirements

**Describe alternatives you've considered**
There aren't really any alternatives, I guess it's something that just needs to be considered and added to the system

**Comment by shilo-ey** [comment]
Hi @dave-agilepixel 

We are taking accessibility issues very seriously, If you have a concrete and specific issue, please create a new feature request regarding it and we will validate and take care of it.

Thanks

**Comment by swinggraphics** [comment]
@shilo-ey I _did_ open a couple of discussions regarding specific accessibility features. You closed [one](https://github.com/elementor/elementor/issues/9292) after providing a non-working workaround. That is not "taking accessibility issues very seriously." It's like adding a ramp for wheelchairs that's too steep to actually use. And we/you can actually be [sued for it](https://www.essentialaccessibility.com/blog/web-accessibility-lawsuits/). Remediating a site in such cases would mean rebuilding it from scratch without Elementor because of its limitations.
